frontstage/exceptions/exceptions.py

```diff
@@ -93,3 +93,7 @@ class UserDoesNotExist(Exception):
     def __init__(self, message):
         super().__init__()
         self.message = message
+
+
+class IncorrectAccountAccessError(Exception):
+        pass
```

Comment:
Why isn't this implemented?

Reply:
It just doesn't need to do anything when it happens other than return the template, which is done elsewhere

Reply:
@mfcrocker Essentially the same as a abstract class - the value is in identifying the error, that said...

@ChristopherMShelton Shouldn't we be adding something to this to log the occurence?

Reply:
We've done this in a bunch of other places in other repos.  I'm fine with it not having logging here, but in the places that we ARE throwing this exception (just the 1 place at the minute), we should be logging an error

EDIT: Basically what owen said ^^